pkg/scalers/redis_streams_scaler.go

```diff
@@ -185,28 +209,34 @@ func parseRedisStreamsMetadata(config *ScalerConfig, parseFn redisAddressParser)
 		meta.connectionInfo.unsafeSsl = parsedVal
 	}
 
-	meta.targetPendingEntriesCount = defaultTargetPendingEntriesCount
```

Comment:
We should maintain the default value as it's documented and it has been there for multiple versions. To remove it we have to follow the deprecation policy

Reply:
Default value does not work in main branch, if metadata has not `pendingEntriesCount` key, the `ErrRedisMissingPendingEntriesCount` error is rised. I'll bring it back and make it work